### PR TITLE
[fix](schema change) reduce memory usage of create a good deal of shadow tablets in schema change process

### PR DESCRIPTION
**Pull request by Lchangliang**
When doing schema change, it will create many shadow tablets. And many TCreateTabletReq will be new. So we can use object pool to reduct they memory usage.

### CODE REVIEW

**Comment by Lchangliang** [comment]
run buildall

**Comment by Lchangliang** [comment]
run buildall

**Comment by Lchangliang** [comment]
run buildall

**Comment by Lchangliang** [comment]
run buildall

**Comment by Lchangliang** [comment]
run p0

**Comment by Lchangliang** [comment]
run buildall

**Review by luwei16 — APPROVED** [review]

**Review by dataroaring — APPROVED** [review]
LGTM

File: fe/fe-core/src/main/java/org/apache/doris/alter/RollupJobV2.java [hunk 1]
```diff
@@ -218,6 +219,7 @@ protected void createRollupReplica() throws AlterCancelException {
         try {
             BinlogConfig binlogConfig = new BinlogConfig(tbl.getBinlogConfig());
             Preconditions.checkState(tbl.getState() == OlapTableState.ROLLUP);
+            Map<Object, Object> objectPool = new HashMap<Object, Object>();
```
Comment (gavinchou) [inline fe/fe-core/src/main/java/org/apache/doris/alter/RollupJobV2.java:222]: It's better to use a concurrent map.

**Pull request merged**